### ISSUE TITLE
Add ViewPool support for NestedRecyclerView

### ISSUE THREAD

**Issue by vivchar** [vivchar/RendererRecyclerViewAdapter#26]
https://proandroiddev.com/optimizing-nested-recyclerview-a9b7830a4ba7

I will add it in future. Currently I have no a time.

**Comment by vivchar** [comment]
in BETA now 
```
maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
implementation 'com.github.vivchar:RendererRecyclerViewAdapter:2.8.0-SNAPSHOT'
```

**Comment by vivchar** [comment]
some documentation:
https://github.com/vivchar/RendererRecyclerViewAdapter/wiki/Quick-start-with-ViewPool

**Comment by vivchar** [comment]
released in v2.8.0
